apm-sniffer/apm-sdk-plugin/ehcache-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/ehcache/v2/define/EhcachePluginInstrumentation.java

```diff
@@ -82,13 +88,41 @@
                 public String getConstructorInterceptor() {
                     return CONSTRUCTOR_CLASS_INTERCEPT_CLASS;
                 }
+            },
+            new ConstructorInterceptPoint() {
+                @Override
+                public ElementMatcher<MethodDescription> getConstructorMatcher() {
+                    return isPrivate().and(takesArgument(0, named("net.sf.ehcache.Cache")));
+                }
+
+                @Override
+                public String getConstructorInterceptor() {
+                    return PRIVATE_CONSTRUCTOR_CLASS_INTERCEPT_CLASS;
+                }
             }
         };
     }
 
     @Override
     public InstanceMethodsInterceptPoint[] getInstanceMethodsInterceptPoints() {
         return new InstanceMethodsInterceptPoint[] {
+            new InstanceMethodsInterceptPoint() {
+                @Override
+                public ElementMatcher<MethodDescription> getMethodsMatcher() {
+                    return named(CACHE_NAME_ENHANCE_METHOD).and(takesArgument(0, String.class));
+                }
+
+                @Override
+                public String getMethodsInterceptor() {
+                    return CACHE_NAME_INTERCEPTOR_CLASS;
+                }
+
+                @Override
+                public boolean isOverrideArgs() {
+                        return false;
+                    }
```

Comment:
Is this a code format error?

Reply:
Yes. Do I need to format and check?

Reply:
Yes, you should. The format check will be more strict soon for the test case.